### PR TITLE
Add Python PVR Flink job

### PR DESCRIPTION
**Pull request by Amar3tto**
The following jenkins jobs have been migrated:

https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Commit/
https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/
https://ci-beam.apache.org/job/beam_PreCommit_Python_PVR_Flink_Phrase/
### How has it been tested?
The job has been tested on Self-hosted GKE runners ubuntu 20.04.
The following test cases are passed:

- by creating pull requests with changes in trigger paths
- by adding a comment to the PR
- by making a push

Screenshot from 2023-07-21 15-39-20
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/beam_PreCommit_Python_PVR_Flink.yml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        uses: gradle/gradle-build-action@v2
+        with:
+          cache-read-only: false
+      - name: Setup Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          install: true
+          driver: 'docker'
+      - name: run Python PVR Flink PreCommit script
+        run: ./gradlew :sdks:python:test-suites:portable:py311:flinkValidatesRunner -PpythonVersion=3.11
```
Comment (andreydevyatkin) [inline .github/workflows/beam_PreCommit_Python_PVR_Flink.yml:119]: We should use the gradle action to run the command in order to add the required params - https://github.com/apache/beam/blob/master/.github/actions/gradle-command-self-hosted-action/action.yml

File: .github/workflows/beam_PreCommit_Python_PVR_Flink.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+jobs:
+  beam_PreCommit_Python_PVR_Flink:
+    name: beam_PreCommit_Python_PVR_Flink
+    if: |
+      github.event_name == 'push' || 
+      github.event_name == 'pull_request_target' || 
+      github.event.comment.body == 'Run Python_PVR_Flink PreCommit' || 
+      github.event_name == 'schedule'
+    runs-on: [self-hosted, ubuntu-20.04]
```
[the remark this answers is not in the document]
Comment (Amar3tto) [reply .github/workflows/beam_PreCommit_Python_PVR_Flink.yml]: Done

File: .github/workflows/beam_PreCommit_Python_PVR_Flink.yml (hunk 1 above)
Comment (Amar3tto) [reply .github/workflows/beam_PreCommit_Python_PVR_Flink.yml:119]: It doesn't work properly with python3.11, so we need to consider not using this action here.

Comment (Amar3tto) [reply .github/workflows/beam_PreCommit_Python_PVR_Flink.yml]: Fixed

**Pull request closed without merging**